### DISCUSSION TITLE
Sandboxing system daemons with /dev/null as HOME

### DISCUSSION THREAD

**Discussion by crocket** [netblue30/firejail#4778] [Q&A]
If HOME is /dev/null, `include anything.local` fails.

**Comment by ghost** [comment]
I can see how this would make sense for local overrides in ~/.config/firejail. Just to be on the same page, does including `foo.local` fails even when foo.local is in `/etc/firejail`?

**Reply by crocket** [reply]
When foo.local is in /etc/firejail, `include foo.local` succeeds.

**Comment by netblue30** [comment]
Why would you do the /dev/null mounting? If you start your server as root, by default firejail mounts an empty temporary filesystem on top of /root directory, and another empty one on /home. Probably that's why you don't find foo.local in your home.

**Reply by crocket** [reply]
I start system daemons as non-root users inside firejail sandbox. HOME for some non-root users is /dev/null in /etc/passwd.
